test/encode.js

```diff
@@ -0,0 +1,18 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import { encode } from '../src/encode'
+
+chai.use( chaiChange )
+
+describe('encode', () => {
+  'use strict'
+
+  it('exists', () => {
+    expect( encode ).to.be.a( 'function' )
+  })
+
+  it('encodes alphanumeric characters into emoji string ', () => {
+    expect(encode( 'w07' )).to.equal( '😭😅🐱' )
+  })
+
```

Comment:
great job guys! Good job learning testing...now that you have this down study up and challenge yourself to do better test coverage in future projects